ffi/src/prove.rs

```diff
@@ -1,27 +1,44 @@
 use crate::utils::{c_char_to_str, c_char_to_vec, vec_to_c_char};
```

Comment:
I feel it's better to organize the files as `zkevm.rs` and `aggregator.rs` where each file contains both its prover and verifier.

Reply:
Re-organize FFI to `zkevm.rs` and `aggregator.rs` in commit [d23c3e7](https://github.com/scroll-tech/scroll-prover/pull/160/commits/d23c3e7892392c4164805abfad820f2949e9b08c).